docroot/modules/contrib/videojs/src/Plugin/Field/FieldFormatter/VideoJsPlayerListFormatter.php

```diff
@@ -51,27 +54,53 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
       $video_uri = $file->getFileUri();
       $video_items[] = Url::fromUri(file_create_url($video_uri));
     }
-    $elements[] = array(
-      '#theme' => 'videojs',
-      '#items' => $video_items,
-      '#player_attributes' => $this->getSettings(),
-      '#attached' => array(
-        'library' => array('videojs/videojs'),
-      ),
-    );
+    $var = $this->getSettings();
+    $extract_extensions = explode(',', $var['extensions']);
+    foreach ($extract_extensions as $key => $value) {
+      if (stripos($video_uri, $value)) {
+        
+        $ext = $value;
+      }
+      else {
+        $ext = 'mp4';
+      }
+    }
+    if ($this->getSetting('annotations') == 0) {
```

Comment:
Can we set a default variable & update only the index inside the condition rather than re-writing the entire array?